application/container-builder/internal/containers/buildpack/builder.go

```diff
@@ -3,43 +3,48 @@ package image
 import (
 	"context"
 	"fmt"
+	"os"
 	"strings"
 
 	pack "github.com/buildpacks/pack/pkg/client"
-	"github.com/hashicorp/go-hclog"
+	"github.com/null-channel/eddington/proto/container"
+	"github.com/rs/zerolog"
 	"github.com/uptrace/bun"
 )
 
-var log = hclog.New(&hclog.LoggerOptions{
-	Name:  "container-builder⚡️",
-	Level: hclog.LevelFromString("INFO"),
-})
+var log = zerolog.New(os.Stdout).With().Timestamp().Str("component", "builder").Logger()
 
 type BuildOpt struct {
+	BuildID   string
 	RepoURL   string
 	ImageName string
 	BuildPack string
 	Builder   string
 }
 
+// BuildRequest is the request to build a container
 type BuildRequest struct {
```

Comment:
IMO. This is not a request. you don't generally give a "status" in the request. the status is something that could be returned but that is it.

If this was me I would pull all DB models out into their own "models" section and don't reuse these models for other things as this gives us the most flexibility with how we store data on disk. it's also then easier to find where your database schema is :)

Reply:
Agreed. I wasn’t sure if it needed to be its own package but this definitely makes sense